<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonsWidget</class>
 <widget class="QWidget" name="ButtonsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>108</width>
    <height>36</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>108</width>
    <height>36</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>108</width>
    <height>36</height>
   </size>
  </property>
  <widget class="QPushButton" name="installButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/download.png</normaloff>:/Icons/download.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>0</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/delete.png</normaloff>:/Icons/delete.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>0</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/run.png</normaloff>:/Icons/run.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
